pillar.example

```diff
@@ -143,3 +143,7 @@ vault:
     ZF5q4h4I33PSGDdSvGXn9UMY5Isjpg==
     =7pIB
     -----END PGP PUBLIC KEY BLOCK-----
+
+  install_from_repo: False  # Set to True to install package rather than extract archive
+  repo: ... # Might specify a specific repo, if not present in map
+  package: vault  # If package would have any other name
```

Comment:
Please observe the yaml formatting [problems](https://results.pre-commit.ci/run/github/87939623/1674488271.Y6c6VFFrQmiXi4IzH-zqpw) covered by `pre-commit`.
The pillar values are self-explanatory, so do not require commenting but please supply an actual example value for `repo`.

Reply:
updated.